### DISCUSSION TITLE
16th open maintainers call

### DISCUSSION THREAD

**Discussion by mxinden** [libp2p/rust-libp2p#3604] [General]
Hi everyone,

The rust-libp2p maintainers will meet tomorrow (Tuesday). See [libp2p calendar](https://calendar.google.com/calendar/embed?src=libp2p.io_0q9682i3te7eanhe9q7ae1c58g%40group.calendar.google.com).

https://meet.jit.si/rust-libp2p-open-maintainers-call

Everyone is welcome to join, listen along, propose topics and join the discussions.

Please post below in case you already want to add an agenda item.

See you all Tomorrow!

**Comment by thomaseizinger** [comment]
I'll be there.
